docs/docs/troubleshooting/README.md

```diff
@@ -75,7 +75,7 @@ Pipedream displays warnings below steps in certain conditions. These warnings do
 
 See the reference on [running asynchronous code on Pipedream](/code/nodejs/async/).
 
-## Pipedream Internal Errors
+## Limit Exceeded Errors
```

Comment:
To keep consistent with the next ones
```suggestion
### Limit Exceeded Errors
```

Reply:
This was intentional. These are a different category of errors that have several sub-headings beneath them.

They were incorrectly described as "Pipedream Internal Errors", where this title is specifically to this type of error.

Converting this title to an h3 would break the hierarchy of other limitations errors beneath (OOM, timeouts, etc).